static/js/info_overlay.js

```diff
@@ -22,99 +22,147 @@ export let toggler;
 
 const markdown_help_rows = [
     {
-        markdown: "*italic*",
+        markdown: "*" + $t({defaultMessage: "italic"}) + "*",
         usage_html: "(or <kbd>Ctrl</kbd>+<kbd>I</kbd>)",
```

Comment:
I think we probably need to translate these `usage_html` lines with `$t_html`. It should be possible to have them all use a common `(or {key_html})` string for translation, and just pass in different values for key_html as a parameter to `$t_html`.